build.sh

```diff
@@ -3,14 +3,13 @@
 # This script is used to build the OpenShift Docker images.
 #
 # OS - Specifies distribution - "rhel7", "rhel8", "rhel9", "centos7", "c8s", "c9s" or "fedora"
-# VERSION - Specifies the image version - (must match with subdirectory in repo)
+# SINGLE_VERSION - Specifies the image version - (must match with subdirectory in repo)
 # VERSIONS - Must be set to a list with possible versions (subdirectories)
 
 set -eE
 [ -n "${DEBUG:-}" ] && set -x
 
 OS=${1-$OS}
-VERSION=${2-$VERSION}
```

Comment:
I do not remember why this is present, but I do not think we need to remove it now that we have made the changes below keeping current behavior right?

Reply:
Fixed by https://github.com/sclorg/container-common-scripts/pull/344/commits/917cd7fa736348dff69dca6d17be4c13715f35a6